### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by stephanieea**
Insomniac changes -

User can see multiple webviews if desired.

User cannot enter recipes if title and url fields are empty.

Timer is adjusted back to minutes (Previously switched to 5 seconds for quicker debugging).

Fix minor errors over load channel.  

@apsitos @gprocell927 🔍

### CODE REVIEW

**Comment by stephanieea** [comment]
Closes #21 #33

File: lib/renderer.js [hunk 1]
```diff
@@ -20,6 +22,13 @@ $('#save-item-btn').on('click', () => {
   sendItem(item, quantity)
 })
 
+const valueCheck = (title, link) => {
```
Comment (gprocell927) [inline lib/renderer.js:25]: Thanks for catching these nitpicky details! 😄

File: lib/renderer.js [hunk 2]
```diff
@@ -55,7 +68,7 @@ const renderRecipes = (recipes) => {
 }
 
 const renderGroceryList = (groceries) => {
-  JSON.parse(groceries).all_Groceries.map(grocery => {
+  JSON.parse(groceries).grocery_list.map(grocery => {
```
Comment (gprocell927) [inline lib/renderer.js:71]: I'm hoping this worked on your end. I had some trouble last night getting this set back up again after merging a branch I was working on with master. I think the issue now is getting the list to render on list window load. I'm not sure how important that is though.

**Review by gprocell927 — APPROVED** [review]

File: lib/renderer.js (hunk 2 above)
Comment (stephanieea) [reply lib/renderer.js:71]: On this branch, the grocery list loads on render- I think this was all it was :)

File: lib/renderer.js (hunk 1 above)
Comment (stephanieea) [reply lib/renderer.js:25]: well after I kept adding empty things from testing, it seemed like a good thing to do, lol :)

File: lib/renderer.js (hunk 2 above)
Comment (StephanieEA) [inline lib/renderer.js]: It does :)

**Review by apsitos — APPROVED** [review]

**Comment by apsitos** [comment]
Thanks for catching all these little details!

**Pull request merged**